examples/micro-ros_addtwoints_service/platformio.ini

```
@@ -0,0 +1,18 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:teensy41]
+platform = teensy
+board = teensy41
```

```suggestion
; [env:teensy41]
; platform = teensy
; board = teensy41
```

Examples should be board agnostic, leaving this lines commented so users can easily modify them.

Same for the rest of `platformio.ini`